Kevin Williams PA1

```diff
@@ -0,0 +1,365 @@
+ 	#addition portion of program 
+ 	add 	$t0, $t1, $t2	#performs addition 
+ 	sw	$t0, resul1 
+ 	 
+ 	li	$v0, 1		#prints the first num 
+ 	la	$a0, ($t1) 
+ 	syscall	 
+ 	 
+ 	li	$v0, 4		#prints "plus" sign 
+ 	la	$a0, adding	 
+ 	syscall 
+ 	 
+ 	li	$v0, 1		#prints second number 
+ 	la	$a0, ($t2) 
+ 	syscall	 
+ 	 
+ 	li	$v0, 4		#prints "equals" sign 
+ 	la	$a0, equals	 
+ 	syscall 
+ 	 
+ 	li	$v0, 1		#prints result number 
+ 	lw	$a0, resul1 
+ 	syscall	 
+ 	 
+ 	#printing spaces 
+ 	li	$v0, 4		 
+ 	la	$a0, spaces	 
+ 	syscall 
+ 	 
+ 	#subtraction portion of program 
+ 	sub 	$t0, $t1, $t2	#performs subtraction 
+ 	sw	$t0, resul2 
+ 	 
+ 	li	$v0, 1		#prints the first num 
+ 	la	$a0, ($t1) 
+ 	syscall	 
+ 	 
+ 	li	$v0, 4		#prints "minus" sign 
+ 	la	$a0, subtrk	 
+ 	syscall 
+ 	 
+ 	li	$v0, 1		#prints second number 
+ 	la	$a0, ($t2) 
+ 	syscall	 
+ 	 
+ 	li	$v0, 4		#prints "equals" sign 
+ 	la	$a0, equals	 
+	syscall 
+ 	 
+ 	li	$v0, 1		#prints result number 
+ 	lw	$a0, resul2 
+ 	syscall	 
+ 	 
+ 	#printing spaces 
+ 	li	$v0, 4		 
+ 	la	$a0, spaces	 
+ 	syscall 
+ 	 
+ 	#multiplication portion of program 
+ 	mult 	$t1, $t2	#performs multiplication 
+ 	mflo	$t0 
+ 	sw	$t0, resul3 
+ 	 
+ 	li	$v0, 1		#prints the first num 
+ 	la	$a0, ($t1) 
+ 	syscall	 
+ 	 
+ 	li	$v0, 4		#prints "multiplication" sign 
+ 	la	$a0, multi	 
+ 	syscall 
+ 	 
+ 	li	$v0, 1		#prints second number 
+ 	la	$a0, ($t2) 
+ 	syscall	 
+ 	 
+ 	li	$v0, 4		#prints "equals" sign 
+ 	la	$a0, equals	 
+ 	syscall 
+ 	 
+ 	li	$v0, 1		#prints result number 
+ 	lw	$a0, resul3 
+ 	syscall	 
+ 	 
+ 	#printing spaces 
+ 	li	$v0, 4		 
+ 	la	$a0, spaces	 
+ 	syscall 
+ 	 
+ 	#division portion of program 
+ 	div 	$t1, $t2	#performs division 
+ 	mflo	$t0 
+ 	mfhi	$t3 
+ 	sw	$t0, divlo 
+ 	sw	$t3, divhi 
+ 	 
+ 	li	$v0, 1		#prints the first num 
+ 	la	$a0, ($t1) 
+ 	syscall	 
+ 	 
+ 	li	$v0, 4		#prints "division" sign 
+ 	la	$a0, divide	 
+ 	syscall 
+ 	 
+ 	li	$v0, 1		#prints second number 
+ 	la	$a0, ($t2) 
+ 	syscall	 
+ 	 
+ 	li	$v0, 4		#prints "equals" sign 
+ 	la	$a0, equals	 
+ 	syscall 
+ 	 
+ 	li	$v0, 1		#prints quotient result number 
+ 	lw	$a0, divlo 
+ 	syscall	 
+ 	 
+ 	li	$v0, 4		#prints "remainder" sign 
+ 	la	$a0, remain	 
+ 	syscall 
+ 	 
+ 	li	$v0, 1		#prints remainder result number 
+ 	lw	$a0, divhi 
+ 	syscall 
+ 	 
+ 	#printing spaces 
+ 	li	$v0, 4		 
+ 	la	$a0, spaces	 
+ 	syscall 
+ 	la	$a0, spaces	 
+ 	syscall 
+ 	 
+ 	#program exit 
+ 	li	$v0, 4		#prints "goodbye message" sign 
+ 	la	$a0, bye 
+ 	syscall 
+  
+ 	li 	$v0, 10		#program exit syscall 
+ 	syscall 
+next file
+#Author : Kevin Williams
+#Data:  02/11/2015
+.data 
+   
+
+  
+.text
+.globl main
+main:
+    addi $t1, $0, 1 
+    addi $t2, $0, 3
+
+loop:
+        bge     $t2, 11, Exit   # check to see if $t3 >= 11 if so exit
+        add    $t1, $t1 $t2,  # add $t2 to $t1    (1+3),....3+5 ect
+        addi $t2, $t2, 2   # change $t2 to next odd number by adding 2
+
+    j loop  #jump back to the start of the loop
```

Comment:
-6 Instructions require to store and retrieve partial sum value from memory each iteration.